build-support/docker/Consul-Dev-Multiarch.dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG CONSUL_IMAGE_VERSION=latest
+FROM consul:${CONSUL_IMAGE_VERSION}
+RUN apk update && apk add iptables
+ARG TARGETARCH
+COPY linux_${TARGETARCH}/consul /bin/consul
```

Comment:
I'm a bit confused about why we need this new dockerfile. Why not use the same dockerfile with a default to the current arch used (amd64 I believe)?

Reply:
I believe I went this route because I didn't know what the default value should be depending on the architecture of the machine. Thinking about it more, maybe we could just change the existing one to always pass `TARGETARCH` when we're not doing a multi-arch build.

Reply:
Reviewing now, the existing file is used in CI, with the consul binary ending up in a different location to copy over. I think there was a lot less risk/effort in creating the new file than trying to unwind what's going in CI.